quicklogic/common/toolchain_wrappers/ql_symbiflow

```diff
@@ -21,7 +21,7 @@ echo -e "\nBelow are the supported commands: \n\
 \t>ql_symbiflow -synth -src <source_dir path> -d <device> -P <pinmap csv file> -t <top module name> -v <verilog file/files> -p <pcf file>\n\
  To run synthesis, pack, place and route:\n\
 \t>ql_symbiflow -compile -src <source_dir path> -d <device> -P <pinmap csv file> -t <top module name> -v <verilog file/files> -p <pcf file> -P <pinmap csv file> -s <SDC file> \n\
-Devices supported:qlf_k4n8, qlf_k6n10 " || exit
+Devices supported: ql-eos-s3, ql-pp3, qlf_k4n8, qlf_k6n10 " || exit
```

Comment:
Actually, we disabled only the CI tests for the `k6n10` but it is a good idea to add such messages. When the issue will be solved it will require only reverting the https://github.com/QuickLogic-Corp/symbiflow-arch-defs/pull/488/commits/a7fc369f556663dac1cd041e8780136c2047fc52